recipes/r-glmtrans/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set version = "2.0.0" %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-glmtrans
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/glmtrans_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/glmtrans/glmtrans_{{ version }}.tar.gz
+  sha256: 43fa44945db364fbf3ad943d66b5303c58e70bc501d458ede0b35b9973f8a981
+
+build:
+  number: 1
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-glmnet
+    - r-ggplot2
+    - r-foreach
+    - r-doparallel
+    - r-caret
+    - r-assertthat
+    - r-formatr
+  run:
+    - r-base
+    - r-glmnet
+    - r-ggplot2
+    - r-foreach
+    - r-doparallel
+    - r-caret
+    - r-assertthat
+    - r-formatr
+
+test:
+  commands:
+    - $R -e "library('glmtrans')"           # [not win]
+    - "\"%R%\" -e \"library('glmtrans')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=glmtrans
+  license: GPL-2.0-or-later
```

[CRAN shows `GPL-2`](https://github.com/cran/glmtrans/blob/27cfd76926c7ff481b64ce82ad56371e15673400/DESCRIPTION#L10), so

```suggestion
  license: GPL-2.0-only
```